lib/iris/experimental/representation.py

```diff
@@ -190,12 +191,14 @@ def _make_header(self):
         """
         # Header row.
         tlc_template = \
-            '<th class="iris iris-word-cell">{self.name} ({self.units})</th>'
-        top_left_cell = tlc_template.format(self=self)
+            '<th class="iris iris-word-cell">{name} ({units})</th>'
+        top_left_cell = tlc_template.format(name=html.escape(self.name),
+                                            units=html.escape(self.units))
```

Comment:
I'd prefer to see both of these (and each `dim_name` below) sanitized in the init. That way if we end up using them somewhere else in the class in the future we won't have to duplicate these `escape` calls.